src/app-layout/visual-refresh/trigger-button.scss

```diff
@@ -51,4 +52,15 @@ handleSplitPanelMaxWidth function in the context.
       background: awsui.$color-background-layout-toggle-selected-active;
     }
   }
+
+  &.badge::after {
+    content: '';
+    position: absolute;
+    width: 12px;
+    height: 12px;
+    border-radius: 8px;
+    top: 0;
+    right: 0;
+    background-color: awsui.$color-text-status-error;
```

Comment:
Don't you find this risky to use semantically different token here? Should we introduce a separate token here to avoid accidental overrides or theming issues?

Reply:
Good point, I will add an additional token.